public/app/features/datasources/utils/passwordHandlers.test.ts

```diff
@@ -2,7 +2,7 @@ import { createResetHandler, PasswordFieldEnum, Ctrl } from './passwordHandlers'
 
 describe('createResetHandler', () => {
   Object.keys(PasswordFieldEnum).forEach(fieldKey => {
-    const field = PasswordFieldEnum[fieldKey];
+    const field: any = PasswordFieldEnum[fieldKey as any];
```

Comment:
Bit weird this complains, seems to be like a sanctioned way to loop over the enum but still could not make it work with better typing.